GDJS/GDJS/Extensions/Builtin/AsyncExtension.cpp

```diff
@@ -68,6 +68,8 @@ AsyncExtension::AsyncExtension() {
         return "{\n" + parentAsyncObjectsListGetter + "{\n" +
                asyncObjectsListBuilder + asyncActionCode + "}\n" + "}\n";
       });
+
+      GetAllActions()["ResolveAsyncEventsFunction"].SetFunctionName("gdjs.evtTools.common.resolveAsyncEventsFunction");
```

Comment:
Not sure what I did wrong but this is not working...

Reply:
The function is never generated in the code?

Reply:
You probably missed the namespace :)